docs/data/material/getting-started/learn/learn.md

```diff
@@ -1,61 +1,65 @@
-# Learn MUI
+# Learning Resources
 
-<p class="description">New to MUI? It's easy to learn if you know where to start! This guide will help you to get started quickly.</p>
+<p class="description">New to Material UI? Get up to speed quickly with our curated list of learning resources.</p>
 
-Developers come to MUI from different backgrounds and with different learning styles. Whether you prefer a more theoretical or practical approach, we hope you'll find this section helpful.
-Like any unfamiliar technology, MUI does have a learning curve. With practice and some patience, you will soon get the hang of it.
+## Your first component
 
-## First example
+The following code snippet contains everything you need to start building an app with Material UI, beginning with the `<Button>` component:
```

Comment:
> contains everything you need

I wonder... does it?! Ever since I read this line (it has been on the documentation for quite some time) it got me thinking. I assume that "everything" refers to the component import and its usage inside a function. For some reason, it doesn't feel like "everything".

Reply:
I think it is exaggerated. It can be something like: `Let's start with a Button!`

Reply:
Good call! "Everything you need" is a bit of an overstatement. I'll come up with a different way to introduce this code snippet instead.